bag_unpacker/src/test/scala/uk/ac/wellcome/platform/archive/bagunpacker/services/s3/S3UnpackerTest.scala

```diff
@@ -98,9 +101,9 @@ class S3UnpackerTest extends UnpackerTestCases[Bucket] with S3Fixtures {
 
           val underlyingError =
             ingestResult.asInstanceOf[IngestFailed[UnpackSummary]]
-          underlyingError.e shouldBe a[Throwable]
+          underlyingError.e shouldBe a[AmazonS3Exception]
           underlyingError.e.getMessage should startWith(
-            "The specified bucket is not valid"
+            "The specified bucket does not exist"
```

Comment:
Should you update the test name to indicate we're looking for existence over failure?

Reply:
If you scroll up, you'll see the test name already says that:

```scala
it("fails if asked to write to a non-existent bucket") {
```

But we'd never noticed it was checking the wrong message!